src/components/PetTabs.js

```diff
@@ -67,7 +67,8 @@ const PetTabs = props => {
                             <List
                                 dataSource={petTabs[selectedTab]}
                                 renderItem={pet => (
-                                    <ProfileCard key={pet.AnimalId} {...pet} />
+                                    <PetDetails key={pet.AnimalId} pet={pet} />
+                                    // <ProfileCard key={pet.AnimalId} {...pet} />
```

Comment:
remove this

Reply:
removed in https://github.com/baltimorecounty/react-animal-services-pets/pull/9/commits/0b3c6198e976ee7267ca0237f54ad87c70988242